src/language/python/Makefile

```diff
@@ -0,0 +1,35 @@
+-include ../../../Make.inc
```

Comment:
Can this be integrated into the main FreeRADIUS build?  I.e. the boilermake framework?

Why does this have to be a separate `Makefile`?

The default should be to build everything, so long as the correct headers / libraries are on the system.

Reply:
Python specific build system

Reply:
I converted to _boiler.mk_ framework as can be seen in `src/language/python/all.mk`